echidna/core/spectra.py

```diff
@@ -108,14 +113,25 @@ def sum(self):
         """
         return self._data.sum()
 
-    def normalise(self, count):
-        """ Normalise the total counts in the spectra to count, i.e. times each
-        bin by count / self.sum().
+    def scale(self, num_decays):
+        """ Scale THIS spectra to represent *num_decays* worth of decays.
+        
+        This rescales each bin by the ratio of *num_decays* to 
+        *self._num_decays*, i.e. it changes the spectra from representing
+        *self._num_decays* to *num_decays*. *self._num_decays* is updated
+        to equal *num_decays* after.
 
         Args:
-          count (float): Total number of events to normalise to.
+          num_decays (float): Number of decays this spectra should represent.
         """
-        numpy.multiply(self._data, count / self.sum())
+        numpy.multiply(self._data, num_decays / self._num_decays)
```

Comment:
I think this still needs to be:

```(python)
self._data = numpy.multiply(self._data, num_decays /self._num_decays)
```

otherwise numpy doesn't save the changes to `self._data`.